### PR TITLE
Portfolio: Popup window - Final

### PR DESCRIPTION
**Pull request by lily4178993**
In this branch, my coding partner [@Mike111222](https://github.com/Mike111222) and I have created a box/popup window that is displayed in the middle of the current page.

- We use a JavaScript array to store all of the information for all projects
  - For each project we stored the following pieces of data in a JavaScript object: name, description, featured image, technologies, link to live version, link to source.
 - We updated the main page so that the projects section is created dynamically using the information stored in that JavaScript object.
 - After that, we implemented the popup window using the [example](https://www.w3schools.com/howto/howto_css_modals.asp) provided by [W3Schools.com](https://www.w3schools.com/). 
    - We modified the code according to this project for both mobile and desktop screen sizes using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471).
   
 I kindly request someone to lend their expertise by examining my branch. I would greatly appreciate any comments provided. Here is the [link](https://lily4178993.github.io/Portfolio/) of the live demonstration of the site.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -374,6 +378,150 @@
...
+  .modal-description p {
+    opacity: 0.6;
+  }
+
+  .modal-buttons {
+    display: flex;
+    justify-content: flex-start;
+    gap: 2em;
+    margin: 0.9em 0;
+  }
```
Comment (franclobo) [inline styles.css:482]: - [ ] Please 🙏🏽 fix the button position in the mobile version of the popup window for a better UI.

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @lily4178993 and @Mike111222, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Good inline comments to describe the code ⭐ ⭐ ⭐ ⭐ ⭐ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (lily4178993) [reply styles.css:482]: Thank you @franclobo for the review. I change the size of the buttons as you requested.

**Comment by alexander16108** [comment]
Hi @lily4178993 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- No linter errors ✔️
- Correct Github flow ✔️
- Descriptive PR ✔️
- Good inline comments to describe the code ⭐ ⭐ ⭐ ⭐ ⭐

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — APPROVED** [review]

**Review by alexander16108 — APPROVED** [review]

**Comment by lily4178993** [comment]
Thank you all for taking the time to review this branch.

**Comment by lily4178993** [comment]
Thank you all for taking the time to review this branch.

**Pull request merged**